components/Navigation/index.js

```diff
@@ -0,0 +1,50 @@
+import Link from "next/link";
+import styled from "styled-components";
+import HomeSvg from "@/public/home.svg";
+
+export default function Navigation() {
+  return (
+    <StyledNavigation>
+      <StyledList>
+        <li>
+          <StyledLink href="/">
+            <StyledSvg />
+          </StyledLink>
+        </li>
+      </StyledList>
+    </StyledNavigation>
+  );
+}
+const StyledNavigation = styled.nav`
+  width: 100%;
+  margin-bottom: auto;
+  background-color: var(--primary-color);
+  color: var(--neutral-color);
+  bottom: 0;
+  color: white;
+  padding: 15px 0;
+  position: fixed;
+  bottom: 0;
+  padding: 15px;
+  display: grid;
+  z-index: 1;
+`;
```

Comment:
- remove duplicate styles 🙂
- ` margin-bottom: auto` I guess it's not typically necessary for a fixed-position element - _pleas check this first_

```suggestion
const StyledNavigation = styled.nav`
  width: 100%;
  background-color: var(--primary-color);
  color: var(--neutral-color);
  position: fixed;
  bottom: 0;
  padding: 15px;
  display: grid;
  z-index: 1;
`;
```

Reply:
You're totally right! We removed them.